components/script/script_thread.rs

```diff
@@ -2143,8 +2083,6 @@ impl ScriptThread {
 
         // https://html.spec.whatwg.org/multipage/#the-end steps 3-4.
         document.process_deferred_scripts();
-
-        window.set_fragment_name(final_url.fragment().map(str::to_owned));
```

Comment:
Hmm, I wonder if the goal of this code was to scroll to the fragment once the load event was dispatched (although I bet that relied on a resize event being processed afterwards, and we probably broke that since this code was written :P)